### PR TITLE
refactor the api/reviews/review_id to include comment_count property

### PR DESCRIPTION
**Pull request by AwaisQarni4**
This pull request refactors the api/reviews/:review_id endpoint to include the comment_count property. It includes reviews with or without any comments associated to them.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
+messaging.sql
```
Comment (alexdevdavis) [inline .gitignore]: this is messing, right?

File: models/models.js [hunk 2]
```diff
@@ -8,8 +8,17 @@ const fetchCategories = () => {
...
+      `SELECT reviews.*, COUNT(comments.comment_id) ::INT AS comment_count
+      FROM reviews 
+      LEFT JOIN comments 
+      ON reviews.review_id = comments.review_id 
+      WHERE reviews.review_id = $1
+      GROUP BY reviews.review_id;`,
+      [id]
+    )
     .then(({ rows }) => {
+      console.log(rows);
```
Comment (alexdevdavis) [inline models/models.js]: floaty log

**Review by alexdevdavis — APPROVED** [review]
Nicely done, see comments for tweaks, but happy to approve this

File: .gitignore (hunk 1 above)
Comment (AwaisQarni4) [reply .gitignore]: yes, will refactor

File: models/models.js (hunk 2 above)
Comment (AwaisQarni4) [reply models/models.js]: gonna remove

**Pull request merged**